torch/quantization/fx/quantize.py

```diff
@@ -266,10 +266,31 @@ def prepare(self, model, qconfig_dict, inplace=False):
     def prepare_dynamic(self, model, qconfig_dict, inplace=False):
         return self._prepare(model, qconfig_dict, inplace, is_dynamic_quant=True)
 
+    # Extract the subgraph that produces the weight for dynamically quantized node
+    # and run the subgraph to observe the weight
+    def _run_weight_observers(self, observed):
+        for node in observed.graph.nodes:
+            if node.op == 'call_function' and node.target in WEIGHT_INDEX_DICT:
```

Comment:
How does this work for dynamic quantizable modules?

Reply:
for dynamic quantizable modules, the weight observer is run during conversion: e.g. https://github.com/pytorch/pytorch/blob/master/torch/nn/quantized/dynamic/modules/linear.py#L96

Reply:
Can we add a comment here which mentions this?

Reply:
sure